psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -570,19 +570,26 @@ void DFHelper::prepare_AO_wK_core() {
     std::pair<size_t, size_t> plargest = pshell_blocks_for_AO_build(memory_, 1, psteps);
 
     wPpq_ = std::unique_ptr<double[]>(new double[big_skips_[nbf_]]);
-    Ppq_ = std::unique_ptr<double[]>(new double[big_skips_[nbf_]]);
     m1Ppq_ = std::unique_ptr<double[]>(new double[big_skips_[nbf_]]);
 
+    if (!wcombine_) Ppq_ = std::unique_ptr<double[]>(new double[big_skips_[nbf_]]);
```

Comment:
Small nitpick: could you switch these constructors to use `make_unique` instead?
```
Ppq_ = std::make_unique<double[]>(big_skips_[nbf_]);
```

Reply:
I will be making the changes. Someone might want to take a look at the rest of dfhelper.cc to make this change to another group of `unique_ptr` 's. I don't know what all the rest of the code does, so I don't think I should submit those changes in this PR.

Reply:
I ended up making changes only to functions touched by this PR. I'll leave the other functions to people who know what to do with them.